comment/templates/comment/content.html

```diff
@@ -1,14 +1,15 @@
 {% load comment_tags %}
+{% load i18n %}
```

Comment:
what the purpose of loading i18n?

Reply:
It was added previously when we had the text(`Like`) in place of icons. I will use a title attribute to the images and put the text inside the `trans` tag. Just to reduce our work when we decide to make the application to be portable to other languages than English.